webapp/portlet/src/main/webapp/external-spaces-list/externalSpacesList.js

```diff
@@ -0,0 +1,50 @@
+import { spacesConstants } from '../js/spacesConstants.js';
+
+
+export function getExternalSpacesList(offset, limit) {
+  return fetch(`${spacesConstants.SOCIAL_USER_API}${eXo.env.portal.userName}/spaces?offset=${offset || 0}&limit=${limit|| 10}&returnSize=true`, {
```

Comment:
```suggestion
  return fetch(`${spacesConstants.SOCIAL_USER_API}${eXo.env.portal.userName}/spaces?offset=${offset || 0}&limit=${limit|| 10}`, {
```
offset and limit to be verified